docs/esi-leap-reporting.md

```diff
@@ -0,0 +1,31 @@
+### Generating a report
+
+Esi-Leap along with the python-esi-leap client allow the user to easily generate reports on usage
+of a resource over time.
+
+To generate a csv file with data on all usage on different nodes:
+```
+openstack lease contract list --all --long -f csv > report.csv
+```
+
+The Esi-Leap api allows users to retrieve data based on certain parameters. 
+
+For example, it is possible to generate a report based on a user's openstack project_id. To get a list of all
```

Comment:
This essentially views a lessee's usage, correct? If so, can we say "based on a lessee's openstack project_id"?

Reply:
fixed